<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWindow</class>
 <widget class="QMainWindow" name="ReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ReportWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;
color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_materials">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Материалы:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_causeOfMalfunction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Причина неисправности: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>580</y>
      <width>88</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_assistanceProvided">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Оказанная помощь: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_price">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоимость:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_price_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_materials">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_causeOfMalfunction">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_assistanceProvided">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_dateStart">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Время_начала: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dateStart_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_dateEnd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Время_конца: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dateEnd_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
